tensorflow/core/kernels/mkl_conv_ops.cc

```diff
@@ -323,7 +323,7 @@ class MklConvFwdPrimitiveFactory : public MklPrimitiveFactory<T> {
       const MklConvFwdParams& convFwdDims, bool do_not_cache) {
     MklConvFwdPrimitive<T, Tinput, Tfilter, Tbias, Toutput>* conv_fwd = nullptr;
 
-    if (do_not_cache) { /* Always create new primitive */
+    if (do_not_cache) {/* Always create new primitive */
```

Comment:
I think the space between `{` and `/*` should stay. Could you please try applying `clang-format`?